<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddForm</class>
 <widget class="QDialog" name="AddForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj zadanie</string>
  </property>
  <property name="action" stdset="0">
   <string>addTask</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tytuł:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTitle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Opis:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtDescription">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="alertBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
font-weight: bold;
font-size: 15px;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPriority">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Priorytet:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="priorityBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Niski</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Średni</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wysoki</string>
    </property>
   </item>
  </widget>
  <zorder>labelTitle</zorder>
  <zorder>cancelButton</zorder>
  <zorder>txtTitle</zorder>
  <zorder>labelDescription</zorder>
  <zorder>addButton</zorder>
  <zorder>alertBox</zorder>
  <zorder>labelPriority</zorder>
  <zorder>priorityBox</zorder>
  <zorder>txtDescription</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
